taichi/runtime/program_impls/vulkan/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ add_library(vulkan_program_impl)
 target_sources(vulkan_program_impl
   PRIVATE
     vulkan_program.cpp
+    vulkan_program.h
```

Comment:
out of curiosity, is it necessary to put header files in `target_sources`?

Reply:
Makes the experience a little better when generating IDE solutions